share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -241,7 +241,15 @@ Check the top of the e.g. `./configure` output. Some configure scripts do not re
 
 Please add a [`test do`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#test-class_method) block to the formula. This will be run by `brew test foo` and the [Brew Test Bot](Brew-Test-Bot.md).
 
-The [`test do`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#test-class_method) block automatically creates and changes to a temporary directory which is deleted after run. You can access this [`Pathname`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Pathname) with the [`testpath`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#testpath-instance_method) function.
+The
+[`test do`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#test-class_method)
+block automatically creates and changes to a temporary directory which
+is deleted after run. You can access this
+[`Pathname`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Pathname)
+with the
+[`testpath`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#testpath-instance_method)
+function.  The environment variable `HOME` is set to `testpath` within
```

Comment:
Sorry, I messed up my comment.

We're using both `environment variable` & `environmental variable` in this document; was musing it'd be nice to make those consistent since we're here tweaking the document. I think I introduced the latter to be honest. I'm just being anal, really.